modules/customize-certificates-add-service-serving-configmap.adoc

```diff
@@ -32,23 +32,24 @@ $ oc annotate configmap <configmap-name> \//<1>
 +
 [NOTE]
 ====
-Explicitly referencing the `service-ca.crt` key in a volumeMount will
-prevent a Pod from starting until the ConfigMap has been injected with
-the CA bundle.
+Explicitly referencing the `service-ca.crt` key in a volumeMount will prevent a Pod from starting until the ConfigMap has been injected with the CA bundle. This behavior can be overridden by setting the `optional` field to `true` for the volume's serving certificate configuration.
```

Comment:
@marun Can you check that the sentence I added for overriding this behavior is correct?

Reply:
lgtm